forc-plugins/forc-crypto/src/args.rs

```diff
@@ -0,0 +1,136 @@
+//! A `forc` plugin for converting a given string or path to their hash.
+
+use std::{
+    fs::read,
+    io::{self, BufRead},
+    path::Path,
+};
+
+#[derive(Debug, Clone, clap::Args)]
+#[clap(author, version, about = "Hashes the argument or file with this hash")]
+pub struct HashArgs {
+    /// This argument is optional, it can be either:
+    ///
+    ///     1. A path to a file. If that is the case, the content of the file is
+    ///        loaded.
+    ///
+    ///     2. A binary string encoded as a hex string. If that is the case, the
+    ///        hex is decoded a Vec<u8> is passed
+    ///
+    ///     3. A string. This is the last option, if the string is "-", `stdin`
```

Comment:
This seems to be how it works, is this intended? (If the string starts with "-")

```suggestion
    ///     3. A string. This is the last option, if the string starts with "-", `stdin`
```

Reply:
No, the string has to be `-`